Calculator-master/script.js

```diff
@@ -0,0 +1,57 @@
+/* jshint esversion: 6 */
+
+const calculatorContainer = document.getElementById('calculatorContainer');
+const numberValue1 = calculatorContainer.querySelector('[data-calculator-value-1]');
+const numberValue2 = calculatorContainer.querySelector('[data-calculator-value-2]');
+const addingOperation = calculatorContainer.querySelector('[data-calculator-operation-adding]');
+const substractiongOperation = calculatorContainer.querySelector('[data-calculator-operation-substraction]');
+const multiplicationOperation = calculatorContainer.querySelector('[data-calculator-operation-multiplication]');
+const divisionOperation = calculatorContainer.querySelector('[data-calculator-operation-division]');
+const resultContainer = calculatorContainer.querySelector('[data-calculator-result-value]');
+const operations = document.getElementsByClassName('calculator__calculate');
+
+addingOperation.addEventListener('click', function () {
+  /** Tu dopisać funkcjonalność dla dodawania */
+
+  const numberValue1 = calculatorContainer.querySelector('[data-calculator-value-1]');
+  const numberValue2 = calculatorContainer.querySelector('[data-calculator-value-2]');
+
+  var sum = 0;
+  sum = number(numberValue1) + number(numberValue2);
```

Comment:
Tu jest błą, nie wiadomo co to `number`. Tu trzeba użyć specjalnej funkcji, która zamienia `string` na `number`. Proszę poszukać :)